.github/pull_request_template.md

```diff
@@ -1,13 +1,110 @@
-# <!-- Use the title to describe PR changes in the imperative mood --> #
+## Ticket
 
-## 🗣 Description ##
+Resolves #00
 
-<!-- Describe the "what" of your changes in detail. -->
-<!-- Please link to any relevant issues. -->
+## Changes
 
-## 💭 Motivation and context ##
+<!-- What was added, updated, or removed in this PR. -->
+- Change 1
+- Change 2
 
-<!-- Why is this change required? -->
-<!-- What problem does this change solve? How did you solve it? -->
-<!-- Mention any related issue(s) here using appropriate keywords such -->
-<!-- as "closes" or "resolves" to auto-close them on merge. -->
\ No newline at end of file
+<!--
+    Please add/remove/edit any of the template below to fit the needs
+    of this specific PR
+--->
+
+## Context for reviewers
+
+<!--Background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers.  -->
+
+## Setup
+
+<!--  Add any steps or code to run in this section to help others run your code.
+    
+    Example 1:
+    ```sh
+    echo "Code goes here"
+    ``` 
+    
+    Example 2: If the PR was to add a new link with a redirect, this section could simply be:
+    -go to /path/to/start/page
+    -click the blue link in the <insert location>
+    -notice user is redirected to <proper end location>
+-->
+
+## Code Review Verification Steps
+
+### As the original developer, I have
+
+#### Satisfied acceptance criteria and met development standards
+
+- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
+- [ ] Created/modified automated tests
+- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
+- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
+- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
+
+* Code standards are met:
+
+- [ ] All new functions and methods are commented using plain language
+- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
+- [ ] Interactions with external systems are wrapped in try/except
+- [ ] Error handling exists for unusual or missing values
+
+#### Validated user-facing changes (if applicable)
+
+- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
+- [ ] Checked keyboard navigability
+- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
+- [ ] Add at least 1 designer as PR reviewer
+
+### As code reviewer(s), I have
```

Comment:
```suggestion
### As a code reviewer, I have
```
This is a bit more consistent in my opinion with whats written for design reviewer (As a designer reviewer, I have) since multiple design reviewers can also exist on a ticket

Reply:
We wont have multiple design reviewers review a ticket usually. They can be assigned, but rarely will they review it. We should almost always have multiple people looking at a single ticket. I can make a note though that multiple code reviewers can share the same checklist